alibi_detect/od/pytorch/knn.py

```diff
@@ -0,0 +1,103 @@
+from typing import Optional, Union, List, Tuple
+
+import numpy as np
+import torch
+
+from alibi_detect.od.pytorch.ensemble import Accumulator
+from alibi_detect.od.pytorch.base import TorchOutlierDetector
+
+
+class KNNTorch(TorchOutlierDetector):
+    def __init__(
+            self,
+            k: Union[np.ndarray, List, Tuple],
+            kernel: Optional[torch.nn.Module] = None,
+            accumulator: Optional[Accumulator] = None,
+            device: Optional[Union[str, torch.device]] = None
+            ):
+        """PyTorch backend for KNN detector.
+
+        Parameters
+        ----------
+        k
+            Number of neirest neighbors to compute distance to. `k` can be a single value or
+            an array of integers. If `k` is a single value the outlier score is the distance/kernel
+            similarity to the `k`-th nearest neighbor. If `k` is a list then it returns the distance/kernel
+            similarity to each of the specified `k` neighbors.
+        kernel
```

Comment:
I think this description could more clearly/directly say that the kernel is used to define the distance between data points

Reply:
e.g. 'we compute the kernel similarity while computing' -> 'the kernel'